pgh_play/www/php/interface.php

```diff
@@ -201,5 +203,72 @@ function get_parks(){
 	return $results;
 }
 
+// enter new park into the db via new park form
+// this will store the following info in the db: Park name, address, rating, ages, comments
+//  all of these inputs are received by the new park form a user fills out
+// *****facilities and pictures still need to be implemented 
+function new_park(){
+	global $db;
+	connect();
+	$park_id = 0;
+	
+	
+	// if name of park is entered, allow user to store name, address, and zip code of the new park
+	if(isset($_POST['name'])){
+		// set neighborhood if user chooses to
+		if(isset($_POST['neighborhood'])){
+			$query = "INSERT INTO parks (name, address, zip_code, neighborhood) VALUES  ('{$_POST['name']}', '{$_POST['address']}', '{$_POST['zip']}', '{$_POST['neighborhood']}')";
+			$result = $db->query($query) or trigger_error(mysql_error()." ".$query);
+		}
+		else{
+			$query = "INSERT INTO parks (name, address, zip_code) VALUES  ('{$_POST['name']}', '{$_POST['address']}', '{$_POST['zip']}')";
+			$result = $db->query($query) or trigger_error(mysql_error()." ".$query);
+		}
+		echo "Thanks!";
+	}
+	else{
+		echo false;
+	}
+	// get park id variable from park table to correctly link the park to its attributes, ratings, comments, etc. (link  parks to other tables in db)
+		$query = "SELECT park_id FROM parks WHERE name='{$_POST['name']}'";
+		$result = $db->query($query) or trigger_error(mysql_error()." ".$query);
+		$temp=$result->fetch_assoc();
+	
+	// store rating if set
+	if(isset($_POST['rating'])){
+		$query = "INSERT INTO ratings (park_id, rating) VALUES ('{$temp['park_id']}', '{$_POST['rating']}')";
+		$result = $db->query($query) or trigger_error(mysql_error(). " ".$query);
+	}
+	
+
+	// store comment if set
+	if(isset($_POST['comment'])){
+		$query = "INSERT INTO comments (park_id, comment) VALUES ('{$temp['park_id']}', '{$_POST['comment']}')";
+		$result = $db->query($query) or trigger_error(mysql_error(). " ".$query);
+	}
+	// store info about appropriate age groups for park if set
+	if (strcmp($_POST['infants'],"yes") == 0){
```

Comment:
I was setting the age attributes under the impression that these particular ones were fixed. Basically not allowing to set their own age ranges, but giving a fair estimate on which ages they deemed appropriate via checkbox. The other attributes would be dynamic, such as swings and tracks etc.